libtiny_tui/src/msg_area/line.rs

```diff
@@ -361,7 +324,7 @@ mod tests {
     fn height_test_5() {
         let mut line = Line::new();
         line.add_text("abcde");
-        for i in 0..6 {
+        for i in 6..0 {
```

Comment:
Is there a reason for this change?

Reply:
The original test doesn't meet the requirement. Now that I look closer at it, this one doesn't either because it's caching the first value. I think it's ok to remove it.

Reply:
Which requirement do you mean?

Reply:
The long, spaceless lines no longer get cut off. 
Before it took "abcde" and rendered it cut off.
Ex. 
When i/width is 1, the string renders as "a" and the line count is 1.
width = 2 -> "ab", still 1 line
width = 3 -> "abc"...

My tests - `test_calculate_height()`- in `input_line.rs` cover cases like this.